src/charts/line/Context.tsx

```diff
@@ -21,12 +25,14 @@ type LineChartProviderProps = {
   children: React.ReactNode;
   data: TLineChartData;
   yRange?: YRangeProp;
+  onCurrentXChange?: (x: number) => void;
```

Comment:
Maybe we should name this `onCurrentIndexChange`?

Reply:
Done